public/testem/testem_connection.js

```diff
@@ -176,6 +189,13 @@ function initSocket(id) {
   socket.on('stop-run', function() {
     sendMessageToParent('stop-run');
   });
+  socket.on('*', function(event) {
+    if (event.data) {
+      var eventName = event.data[0];
+      var eventData = event.data[1];
+      sendMessageToParent(eventName, eventData);
```

Comment:
I think we should consider namespacing this event, so as to not conflict with other events that application may produce.

Ultimately, I think testem should take this approach for all messages it posts to window